src/content/language/collections.md

```diff
@@ -242,78 +250,180 @@ final constantMap = const {2: 'helium', 10: 'neon', 18: 'argon'};
 
-For example, you can use the spread operator (`...`) to insert
-all the values of a list into another list:
+### Spread operations
+
+With a spread operation, you can
+add values from one collection to another collection. Null
+values inside of a  collection can be passed in, but null
+collections produce an error. A spread operation has this
+signature in a collection:
 
-<?code-excerpt "misc/test/language_tour/built_in_types_test.dart (list-spread)"?>
 ```dart
-var list = [1, 2, 3];
-var list2 = [0, ...list];
-assert(list2.length == 4);
+...<collection_name>
 ```
 
-If the expression to the right of the spread operator might be null,
-you can avoid exceptions by using a null-aware spread operator (`...?`):
+In the following example, the values in `a` are added
+as values in `items`.
 
-<?code-excerpt "misc/test/language_tour/built_in_types_test.dart (list-null-spread)"?>
+<?code-excerpt "misc/lib/language_tour/misc/spread_operator_in_collection_a.dart (code_sample)"?>
 ```dart
-var list2 = [0, ...?list];
-assert(list2.length == 1);
+var a = [1, 2, null, 4];
+var items = [0, ...a, 5]; // [0, 1, 2, null, 4, 5]
 ```
 
-For more details and examples of using the spread operator, see the
-[spread operator proposal.][spread proposal]
+In the following example, an error is produced because
+`a` is null and can't be passed into `items`.
+
+```dart
+var a = null;
+var items = [0, ...a, 4]; // Error
+```
+
+To learn more about the operator used in spread operations
+for collections, see [Spread operator][].
+
+[Spread operator]: /language/operators/#spread-operators
+
+### Null-aware spread operations
+
+With a null-aware spread operation, you can add values from
+one collection to another collection. Null values inside of
+a collection can be passed in, but null collections are
+ignored. A null-aware spread operation has this
+signature in a collection:
+
+```dart
+...?<collection_name>
+```
+
+In the following example, `a` is ignored because it's
+null, but `b` is passed into `items`.
+
+```dart
+var a = null;
+var b = [1, 2, null];
+var items = [0, ...?a, ...?b, 4]; // [0, 1, 2, null, 4]
+```
+
+To learn more about the operator used in null-aware spread
+operations for collections, see [Spread operator][].
+
+[Spread operator]: /language/operators/#spread-operators
 
 <a id="collection-operators"></a>
-### Control-flow operators
 
-Dart offers **collection if** and **collection for** for use in list, map,
-and set literals. You can use these operators to build collections using
-conditionals (`if`) and repetition (`for`).
+### If statements
+
+You can use an `if` statement inside of a collection to
+conditionally include values in the collection based on a
+boolean expression. An `if` statement has this signature in
+a collection:
 
-Here's an example of using **collection if**
-to create a list with three or four items in it:
+```dart
+// boolean expression must be true to include result
+if (<bool_expression>) <result>
```

Comment:
I think this might be clearer if the example was wrapped in a collection literal:

```dart
var list = [
  if (<bool_expression>) <result>
];
```

Reply:
I decided to leave this one out because it calls out one collection type (I'm trying to avoid that with the generic syntax). If you feel strongly, I could create tabs that show each collection type and what this syntax looks like. 
@munificent or @parlough let me know your thoughts.

Reply:
Tentatively leaving as is & closing. If anyone has inspiration for change, re-open and add your thoughts.